lib/mongodb.js

```diff
@@ -275,6 +310,9 @@ MongoDB.prototype.create = function(model, data, options, callback) {
     data._id = oid; // Set it to _id
     idName !== '_id' && delete data[idName];
   }
+
+  data = self.toDatabase(model, data);
```

Comment:
This `toDatabase` is implemented by this PR (line 217). It looks like the user is massaging the data into a [usable format for mongodb](https://docs.mongodb.org/manual/reference/command/geoNear/#command-syntax). We didn't have it before before because we didn't need to pass in the coordinate pair before so it LGTM. @raymondfeng should have the final say though.

@raymondfeng ^